tests/dvslib/dvs_mirror.py

```diff
@@ -0,0 +1,100 @@
+class DVSMirror(object):
+    def __init__(self, adb, cdb, sdb, cntrdb, appdb):
+        self.asic_db = adb
+        self.config_db = cdb
+        self.state_db = sdb
+        self.counters_db = cntrdb
+        self.app_db = appdb
+
+    def create_span_session(self, name, dst_port, src_ports=None, direction="BOTH", queue=None, policer=None):
+        mirror_entry = {"type": "SPAN"}
+        if dst_port:
+            mirror_entry["dst_port"] = dst_port
+
+        if src_ports:
+            mirror_entry["src_port"] = src_ports
+        # set direction without source port to uncover any swss issues.
+        mirror_entry["direction"] = direction
+
+        if queue:
+            mirror_entry["queue"] = queue
+        if policer:
+            mirror_entry["policer"] = policer
+        self.config_db.create_entry("MIRROR_SESSION", name, mirror_entry)
+
+    def create_erspan_session(self, name, src, dst, gre, dscp, ttl, queue, policer=None, src_ports=None, direction="BOTH"):
+        mirror_entry = {"type": "ERSPAN"}
```

Comment:
Do we have a way of testing that orchagent defaults to ERSPAN if no type is specified in the Config DB?

Reply:
Modified to not set type to ERSPAN.